src/domManipulation.ts

```diff
@@ -121,33 +120,34 @@ export function loadWeatherForecast(location: string, fetchResults: Forecast[])
 
       const dateResult = forecastItemContent.substring(
         4,
-        forecastItemContent.indexOf(","),
+        forecastItemContent.indexOf(",")
       );
       forecastItemContent = forecastItemContent.substring(
         forecastItemContent.indexOf(",") + 1,
-        forecastItemContent.length,
+        forecastItemContent.length
       );
       const weatherResult = forecastItemContent.substring(
         0,
-        forecastItemContent.indexOf(","),
+        forecastItemContent.indexOf(",")
       );
       forecastItemContent = forecastItemContent.substring(
         forecastItemContent.indexOf(",") + 1,
-        forecastItemContent.length,
+        forecastItemContent.length
       );
       const maxTempResult = forecastItemContent.substring(
         0,
-        forecastItemContent.indexOf(","),
+        forecastItemContent.indexOf(",")
       );
       forecastItemContent = forecastItemContent.substring(
         forecastItemContent.indexOf(",") + 1,
-        forecastItemContent.length,
+        forecastItemContent.length
       );
       const minTempResult = forecastItemContent.substring(
         0,
-        forecastItemContent.indexOf(","),
+        forecastItemContent.indexOf(",")
       );
-      const forecastItemsList = dailyForecast.getElementsByClassName("forecast-item-info");
+      const forecastItemsList =
+        dailyForecast.getElementsByClassName("forecast-item-info");
 
```

Comment:
little confused about how you are accessing the data. Why are you looping through `result` but not accessing the elements through `result.min` methods, instead of using substrings? It will be easier to do through types than substrings, as well as more readable

Reply:
This was my bad, I was still doing the DOM manipulation the way i was doing it for the initial network call. So I was adding everything into a sentence and getting substrings from this sentence to add to the elements. I have updated this and added comments to explain what's happening.